launch/home/etc/services.cfg

```diff
@@ -20,8 +20,8 @@ org.openhab.autoapprove:enabled=false
 org.eclipse.smarthome.binding.sonos:opmlPartnerID=IAeIhU42
 
 org.jupnp:multicastResponsePort=0
-org.jupnp:streamListenPort=8080
-org.jupnp:threadPoolSize=200
+org.jupnp:asyncThreadPoolSize=30
+org.jupnp:threadPoolSize=15
 
```

Comment:
No, it is a pretty good setting.
Note that up to now, we had 200 threads that were always started at startup. Now we have pools only temporarily increase if necessary, up to a maximum of 45 threads. (and when running a upnp discovery, you can easily have 500-1000 incoming requests to process)